TLM/TLM/UI/NewUI/CanvasForm.cs

```diff
@@ -208,5 +206,27 @@ public class UnityCanvas {
                 false);
```

Comment:
Line above could be simplified using null coalescing operator?

```cs
parent?.transform ?? formObject_.transform,
```

Reply:
Unity docs say that null check does work differently for  GameObjects
https://answers.unity.com/questions/1236014/-operator-not-working.html
Also Rider does not suggest it for this reason i think, otherwise i'd apply the change.